fake-ipa/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG BASE_IMAGE=python:3.9-slim
+FROM $BASE_IMAGE
+ENV PBR_VERSION=1.0.0
+COPY . /root/fake-ipa/
+WORKDIR /root/fake-ipa/
+ENV CONFIG ${CONFIG:-/root/fake-ipa/conf.py}
+RUN pip install -r requirements.txt
```

Comment:
nit: this should be python3 -m pip install

Reply:
done